docs/Installation.md

```diff
@@ -487,15 +371,182 @@ Check the [declaration of prerequisites] to make sure your preferred combination
+> load.
+
+> All binaries and Perl bindings are already installed.
+
+Configure Zonemaster::Backend to use the correct database engine and database
+path:
+
+```sh
+sed -i '' '/[[:<:]]engine[[:>:]]/ s/=.*/= SQLite/' /usr/local/etc/zonemaster/backend_config.ini
+sed -i '' '/[[:<:]]database_file[[:>:]]/ s:=.*:= /var/db/zonemaster/db.sqlite:' /usr/local/etc/zonemaster/backend_config.ini
+```
+
+Create database directory, set correct ownership and create database:
+
+```sh
+cd `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")'`
+install -v -m 755 -o zonemaster -g zonemaster -d /var/db/zonemaster
+env ZONEMASTER_BACKEND_CONFIG_FILE=/usr/local/etc/zonemaster/backend_config.ini su -m zonemaster -c "perl create_db_sqlite.pl"
+```
+
+> SQLite will not run as a daemon and does not need to be started.
+
+> **Note:** See the [backend configuration] documentation for details.
+
+#### 5.2.2 Instructions for other engines (FreeBSD)
+
+See appendices for [MariaDB][MariaDB instructions for FreeBSD] and
+[PostgreSQL][PostgreSQL instructions for FreeBSD].
+
+
+### 5.3 Service startup (FreeBSD)
+
+Enable services at startup:
+
+```sh
+sysrc zm_rpcapi_enable="YES"
+sysrc zm_testagent_enable="YES"
+```
+
+Start services:
+
+```sh
+service zm_rpcapi start
+service zm_testagent start
+```
+
+### 5.4 Post-installation (FreeBSD)
+
+To check the running daemons run:
+
+```sh
+service mysql-server status      # If mysql-server is installed
+```
+```sh
+service postgresql status        # If postgresql is installed
+```
+```sh
+service zm_rpcapi status
+service zm_testagent status
+```
+
+See the [post-installation] section for post-installation matters.
+
+
+## 6. Post-installation
+
+### 6.1 Smoke test
+
+If you have followed the installation instructions for Zonemaster::Backend above,
+you should be able to use the API on localhost port 5000 as below.
+
+```sh
+zmtest zonemaster.net
+```
+
+The command is expected to immediately print out a testid,
+followed by a percentage ticking up from 0% to 100%.
+Once the number reaches 100% a JSON object is printed and zmtest terminates.
+
+
+### 6.2. What to do next?
+
+* For a web interface, follow the [Zonemaster::GUI installation] instructions.
+* For a command line interface, follow the [Zonemaster::CLI installation] instruction.
+* For a JSON-RPC API, see the Zonemaster::Backend [JSON-RPC API] documentation.
+
+
+## 7. Upgrade Zonemaster database
+
+If you upgrade your Zonemaster installation with a newer version of
+Zonemaster-Backend and keep the database, then you might have to upgrade the
+database to use it with the new version of Zonemaster-Backend. Please see the
+[upgrade][README.md-upgrade] information.
+
+For SQLite database upgrading is not needed as of now.
+
+
+
+# Appendices
+
+
+## A. Installation with MariaDB
+
+* [CentOS](#a1-mariadb-installation-for-centos)
+* [Debian/Ubuntu](#a2-mariadb-installation-for-debian-and-ubuntu)
+* [FreeBSD](#a3-mysql-installation-for-freebsd)

```

Comment:
It is better to move to table of contents. And the same below.

Reply:
Doing so will add a sublevel to the table of content that was not present. Should I then add `4.2.1`, `4.2.2`, ...? I would argue against overloading the table of content. That is why I added these links here and not in the main table of content.

Reply:
No, with my suggestion "MariaDB installation on CentOS" would be 8.1.